python/cudf_polars/cudf_polars/experimental/dask_serialize.py

```diff
@@ -102,3 +103,6 @@ def _(header: ColumnHeader, frames: tuple[memoryview, memoryview]) -> Column:
             # Copy the second frame (the gpudata in host memory) back to the gpu
             frames = frames[0], plc.gpumemoryview(rmm.DeviceBuffer.to_device(frames[1]))
             return Column.deserialize(header, frames)
+
+    # Register serializer for spillable DataFrames.
+    register_dask_serialize()
```

Comment:
```suggestion
    # Register rapidsmp serializer if it's installed.
    try:
        from rapidsmp.integrations.dask.spilling import register_dask_serialize

        register_dask_serialize()
    except ImportError:
        pass
```
With top-level import removed.